fearless/Common/Model/ChainModel.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import RobinHood
+
+struct ChainModel: Codable, Equatable {
+    var chainId: String
+    var name: String
+    var externalApi: ChainExternalApi?
+
+    static func == (lhs: ChainModel, rhs: ChainModel) -> Bool {
+        lhs.chainId == rhs.chainId && lhs.externalApi == rhs.externalApi
+    }
+}
+
+struct ChainExternalApi: Codable, Equatable {
+    var staking: ChainExternalApiObject
+    var history: ChainExternalApiObject
+
+    static func == (lhs: ChainExternalApi, rhs: ChainExternalApi) -> Bool {
+        lhs.staking == rhs.staking && lhs.history == rhs.history
+    }
+}
+
+struct ChainExternalApiObject: Codable {
+    var type: String
+    var url: String
```

Comment:
since we're on iOS, please if it's string, call it urlString, url can be named only if NSURL instance